staging/src/github.com/kubeedge/beehive/pkg/core/core.go

```diff
@@ -28,7 +28,7 @@ func StartModules() {
 // GracefulShutdown is if it gets the special signals it does modules cleanup
 func GracefulShutdown() {
 	c := make(chan os.Signal)
-	signal.Notify(c, syscall.SIGINT, syscall.SIGHUP, syscall.SIGTERM, syscall.SIGKILL,
```

Comment:
I see someone say SIGKILL can not be caught by process. The original code here about `SIGKILL` is useless?

Reply:
Yep, just see doc in this [comment](https://github.com/kubeedge/kubeedge/pull/2011#issue-458903971).